flink-table/flink-table-runtime-blink/src/main/java/org/apache/flink/table/runtime/typeutils/WindowKeySerializer.java

```diff
@@ -109,10 +106,9 @@ public void copy(DataInputView source, DataOutputView target) throws IOException
     @Override
     public int serializeToPages(WindowKey record, AbstractPagedOutputView target)
             throws IOException {
-        int windowSkip = checkSkipWriteForWindowPart(target);
         target.writeLong(record.getWindow());
-        int keySkip = keySerializer.serializeToPages(record.getKey(), target);
-        return windowSkip + keySkip;
+        keySerializer.serializeToPages(record.getKey(), target);
+        return 0;
```

Comment:
Here I separate the serialization of the two fields, so actually, no offset now.

Reply:
IIRC, there were tests failed if we do not skip bytes, e.g. the remaining space is not enough for window key.

Reply:
It is OK, `keySerializer.serializeToPages` will skip if need.

Reply:
Return the num of bytes skipped by `keySerializer`?

Reply:
No, we cannot, The return value is to help better relocate the start offset where the data is located, and the offset we need here is the offset that we started to write. Consider this case:
|---First segment----|Second Segment| 
|--------Left 10 bytes|-----------------|
In fact, we will write 8 bytes in the first segment and skip the next two bytes. At this time, its offset should also be 0

Reply:
Actually, the return value is just for save `checkSkipReadForFixLengthPart` in the `mapFromPages`, the cost is very small.

Reply:
Added comments.